### PR TITLE
draft of October Newsletter

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/blog/newsletter-2018-october.rst [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+The general consensus was to strike a balance by including the prompt character in the code samples, but preventing it from being copied and pasted with a little bit of CSS wizardry. You can see examples of how this looks in `Stripe's documentation <https://stripe.com/docs/building-extensions#use-api>`_. And `this blog post <https://evanhahn.com/how-to-disable-copy-paste-on-your-website/>`_ covers a handful of approaches to making text unselectable.
+
+*If you're ever struggling with code sample styling in your own docs, pop into* `#documenting-apis <https://writethedocs.slack.com/messages/documenting-apis>`_
+
+--------------------------------------------------------
+Resources for planning out your information architecture
+--------------------------------------------------------
+
+Oftentimes, when tackling a new docs project, one of the biggest challenges can be figuring out how to approach your information architecture. This month, a user came to the WTD hivemind for help with this very problem. Many of the good documentarians of Slack jumped in to share an excellent array of IA and user testing resources:
```
Comment (Bradamant3) [inline docs/blog/newsletter-2018-october.rst]: s/Oftentimes/Often ?

File: docs/blog/newsletter-2018-october.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+* Built-in sprint planning tools
+
+Challenges:
+
+* Viewing all documentation-related tasks at once
+* Developer tasks and documentation tasks not being one-to-one
+
+Of course, there are those of us who use non-developer driven task management tools – either instead of or in addition to those tools – such as Trello, Asana, or Flow.
+
+While we may use different tools, all that matters in the end is that you utilize a task management tool (or tool set) that works for you!
```
Comment (Bradamant3) [inline docs/blog/newsletter-2018-october.rst]: s/all that/what

File: docs/blog/newsletter-2018-october.rst [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+Challenges:
+
+* Viewing all documentation-related tasks at once
+* Developer tasks and documentation tasks not being one-to-one
+
+Of course, there are those of us who use non-developer driven task management tools – either instead of or in addition to those tools – such as Trello, Asana, or Flow.
+
+While we may use different tools, all that matters in the end is that you utilize a task management tool (or tool set) that works for you! 
+
+*Have more thoughts and feelings about your toolset? Come share them in* `#doctools <https://writethedocs.slack.com/messages/doctools>`_ *or one of the many tool-specific channels we have on Slack.*
```
Comment (Bradamant3) [inline docs/blog/newsletter-2018-october.rst]: rm "and feelings"?

**Review by Bradamant3 — COMMENTED** [review]
A couple of suggestions/nits. Overall, LGTM!

**Review by Bradamant3 — APPROVED** [review]
Gah, silly GH

**Comment by kelly-o** [comment]
awesome, changes made @Bradamant3 🙂

**Pull request merged**